tests/bluetooth/tester/src/btp_csis.c

```diff
@@ -0,0 +1,101 @@
+/* btp_csis.c - Bluetooth CSIS Tester */
+
+/*
+ * Copyright (c) 2023 Oticon
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/bluetooth/audio/csip.h>
+
+#include "btp/btp.h"
+#include "zephyr/sys/byteorder.h"
+
+#include <zephyr/logging/log.h>
+#define LOG_MODULE_NAME bttester_csis
+LOG_MODULE_REGISTER(LOG_MODULE_NAME, CONFIG_BTTESTER_LOG_LEVEL);
+
+static struct bt_csip_set_member_svc_inst *csis_svc_inst;
+
+static uint8_t csis_supported_commands(const void *cmd, uint16_t cmd_len,
+				       void *rsp, uint16_t *rsp_len)
+{
+	struct btp_csis_read_supported_commands_rp *rp = rsp;
+
+	/* octet 0 */
+	tester_set_bit(rp->data, BTP_CSIS_READ_SUPPORTED_COMMANDS);
+	tester_set_bit(rp->data, BTP_CSIS_SET_MEMBER_LOCK);
+	tester_set_bit(rp->data, BTP_CSIS_GET_MEMBER_RSI);
+
+	*rsp_len = sizeof(*rp) + 1;
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t csis_set_member_lock(const void *cmd, uint16_t cmd_len,
+				    void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_csis_set_member_lock_cmd *cp = cmd;
+	int err = -1;
+
+	if (csis_svc_inst) {
+		err = bt_csip_set_member_lock(csis_svc_inst, cp->lock, cp->force);
+	}
+	return (err) ? BTP_STATUS_FAILED : BTP_STATUS_SUCCESS;
+}
+
+static uint8_t csis_get_member_rsi(const void *cmd, uint16_t cmd_len,
+				   void *rsp, uint16_t *rsp_len)
+{
+	struct btp_csis_get_member_rsi_rp *rp = rsp;
+	int err = -1;
+
+	if (csis_svc_inst) {
+		err = bt_csip_set_member_generate_rsi(csis_svc_inst, rp->rsi);
+	}
+	rp->err = sys_cpu_to_le32(err);
```

Comment:
```suggestion
	}
	
	rp->err = sys_cpu_to_le32(err);
```
Minor: We usually keep an empty line after `}` unless that line also contains a `}`

Reply:
Done,